Apps/WebClient/src/Server/Models/UserPreferenceModel.cs

```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Models
+{
+    using System;
+
+    /// <summary>
+    /// Model that provides a user representation of an user preference database model.
+    /// </summary>
+    public class UserPreferenceModel
```

Comment:
Clases like these are only necessary if the db model is different from the one requested at the controller level